src/__tests__/CompareResults/Retrigger.test.tsx

```diff
@@ -0,0 +1,181 @@
+import userEvent from '@testing-library/user-event';
+
+import RetriggerButton from '../../components/CompareResults/Retrigger/RetriggerButton';
+import { getLocationOrigin } from '../../utils/location';
+import getTestData from '../utils/fixtures';
+import {
+  FetchMockSandbox,
+  fireEvent,
+  render,
+  screen,
+  within,
+} from '../utils/test-utils';
+
+jest.mock('../../utils/location');
+const mockedGetLocationOrigin = getLocationOrigin as jest.Mock;
+const result = getTestData().testCompareData[0];
+
+const setUpUserCredentions = () => {
+  mockedGetLocationOrigin.mockImplementation(() => 'http://localhost:3000');
+  sessionStorage.setItem('requestState', 'OkCrH5isZncYqeJbRDelN');
+  sessionStorage.setItem(
+    'taskclusterUrl',
+    'https://firefox-ci-tc.services.mozilla.com',
+  );
+  global.localStorage.setItem(
+    'userTokens',
+    JSON.stringify({
+      'https://firefox-ci-tc.services.mozilla.com': {
+        access_token: 'RnVpOGJtdDZTb3FlWW5PVUxVclprQQ==',
+        token_type: 'Bearer',
+      },
+    }),
+  );
+  global.localStorage.setItem(
+    'userCredentials',
+    JSON.stringify({
+      'https://firefox-ci-tc.services.mozilla.com': {
+        expires: '2044-05-20T14:07:40.828Z',
+        credentials: {
+          clientId:
+            'mozilla-auth0/ad|Mozilla-LDAP|ldapuser/perfcompare-localhost-3000-client-OCvzh5',
+          accessToken: 'jQWJVQdeRceT-YymPwTWagPh2PwJr0RmmZyL1uAfMSWg',
+        },
+      },
+    }),
+  );
+};
+
+describe('Retrigger', () => {
+  it('should display Sign In modal when there are no credentials', async () => {
+    (window.fetch as FetchMockSandbox).get('begin:https://somethingurl', {
+      ceva: 'response',
+    });
```

Comment:
it this useful?

Reply:
nope, I forgot to remove it from this test. Thanks!